security-malware-firewall.php

```diff
@@ -2,7 +2,7 @@
 /*
 Plugin Name: Security by CleanTalk
 Plugin URI: https://wordpress.org/plugins/security-malware-firewall/
-Description: Security & Malware scan by CleanTalk to protect your website from online threats and viruses. IP/Country FireWall, Web application FireWall. Detailed stats and logs to have full control.
+Description: Security & Malware Scanner by CleanTalk for protection of your website from online threats and viruses. IP/Country FireWall, Web Application FireWall. Detailed stats and logs for a full control.
```

Comment:
The title shouldn't be changed. SEO reasons. We need the additional permission to edit it.

Reply:
Done, reverted.